troposphere/awslambda.py

```diff
@@ -201,3 +202,34 @@ class Version(AWSObject):
         'Description': (basestring, False),
         'FunctionName': (basestring, True),
     }
+
+
+class Content(AWSProperty):
+    props = {
+        'S3Bucket': (basestring, True),
+        'S3Key': (basestring, True),
+        'S3ObjectVersion': (basestring, False),
+    }
+
+
+class LayerVersion(AWSObject):
+    resource_type = "AWS::Lambda::LayerVersion"
+
+    props = {
+        'CompatibleRuntimes': (basestring, False),
```

Comment:
This should be an list of basestring per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-layerversion.html

Reply:
@dkavanagh thank you for the review. I just pushed a fix.